extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -0,0 +1,257 @@
+  ],
+  "$id":
+    "https://ns.adobe.com/xdm-extensions/adobe/experience/campaign/experienceevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Campaign ExperienceEvent Extension",
+  "type": "object",
+  "meta:extends": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Adobe Campaign extension to ExperienceEvent.",
+  "definitions": {
+    "experienceevent": {
+      "properties": {
+        "https://ns.adobe.com/experience/campaign/message/id": {
+          "title": "Message Identifier",
+          "type": "number",
+          "description": "Identifier of the message."
+        },
+        "https://ns.adobe.com/experience/campaign/message/profile": {
+          "title": "Profile",
+          "$ref": "https://ns.adobe.com/xdm/context/profile",
+          "description":
+            "The recipient of the message. This property is primarily used to link the message to a Profile using the `EndCustomerID` but it can also be used to freeze some properties of the profile at the time the message was sent."
+        },
+        "https://ns.adobe.com/experience/campaign/message/variant": {
+          "title": "Content Variant",
+          "type": "string",
+          "description":
+            "A campaign activity can have multiple variants (Multilingual or A/B testing) but a message delivered to a recipient always contains one of them.\n> IMPORTANT: In case of multilingual, the mirror page, the recipient can switch from one variant to another (ex: switch of the language)."
+        },
+        "https://ns.adobe.com/experience/campaign/message/seedMember": {
+          "title": "Seed Member",
+          "type": "boolean",
+          "default": false,
+          "description":
+            "The recipient of this message is a member of a seed list for this campaign activity. Seed addresses are used to target recipients who do not match the defined target criteria. Seed members could be your marketing team or other stakeholders who are interested in tracking/measuring the quality of the campaign."
+        },
+        "https://ns.adobe.com/experience/campaign/message/quarantine": {
+          "title": "Quarantine",
+          "type": "boolean",
+          "default": false,
+          "description":
+            "Address moved to quarantine. The quarantine is set to true when the message failed to deliver in a way that indicates that no future messages will be deliverable either to this address."
+        },
+        "https://ns.adobe.com/experience/campaign/message/proofMember": {
+          "title": "Proof Member",
+          "type": "boolean",
+          "description":
+            "The recipient of this message is a member of the proof group for this campaign activity. The proof is a special message that lets you test a delivery before sending to the main target. Proof recipients are responsible for approving both the form and content of the message.",
```

Comment:
Do I get this right:
- `proofMember` gets the proof message, which is a pre-delivery
- `seedMember` gets the actual message, at the same time with regular recipients, even when the `seedMember` does not qualify to be a regular recipient?

If yes, include this in the description.